specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -197,7 +330,7 @@
     },
     "JsonFormatFilePattern": {
       "description": "JSON format file pattern. A property of JsonFormat.",
-      "type": "string",
+      "type": "object",
```

Comment:
breaking change in a stable version?
Any reason this cannot be done in the next API version?

Reply:
Ok, I invited Hermine to explain this change, please wait a moment.

Reply:
@shahabhijeet the type is changing from string to object, so it's not API breaking change as our service will accept both.
We could mark the SDK-s with breaking tag if you think it's appropriate, however we had another PR like this that wasn't marked.

Reply:
@hvermis it is a binary breaking change.
I was concerned about a scenario where your customer has written code against your model as
```
JsonFormat.FilePattern.ToLower().StartWith("array")
```
Also what I noticed that you have modeled your enum as "modelAsString", but you would like to make a change from string to object?

Reply:
@shahabhijeet We'll mark the SDK release as breaking.